library/src/main/java/com/firebase/ui/FirebaseRecyclerViewAdapter.java

```diff
@@ -175,11 +175,39 @@ public VH onCreateViewHolder(ViewGroup parent, int viewType) {
         }
     }
     @Override
-    public void onBindViewHolder(VH viewHolder, int i) {
-        T model = getItem(i);
-        populateViewHolder(viewHolder, model);
+    public void onBindViewHolder(VH viewHolder, int position) {
+        T model = getItem(position);
+        populateViewHolder(viewHolder, model, position);
     }
 
-    abstract public void populateViewHolder(VH viewHolder, T model);
+    /**
+     * Each time the data at the given Firebase location changes, this method will be called for each item that needs
+     * to be displayed. The first two arguments correspond to the mLayout and mModelClass given to the constructor of
+     * this class. The third argument is the item's position in the list.
+     * <p>
+     * Your implementation should populate the view using the data contained in the model.
+     * You should implement either this method or the other FirebaseRecyclerViewAdapter#populateViewHolder(VH, Object) method
+     * but not both.
+     *
+     * @param viewHolder The view to populate
+     * @param model      The object containing the data used to populate the view
+     * @param position  The position in the list of the view being populated
+     */
+    protected void populateViewHolder(VH viewHolder, T model, int position) {
+        populateViewHolder(viewHolder, model);
+    };
+    /**
+     * This is a backwards compatible version of populateViewHolder.
+     * <p>
+     * You should implement either this method or the other FirebaseRecyclerViewAdapter#populateViewHolder(VH, T, int) method
+     * but not both.
+     *
+     * @see FirebaseListAdapter#populateView(View, Object, int)
+     *
+     * @param viewHolder The view to populate
+     * @param model      The object containing the data used to populate the view
+     */
+    protected void populateViewHolder(VH viewHolder, T model) {
```

Comment:
Mark this overload as deprecated, so that devs know to use the overload that takes an aditional `position` param.